### ISSUE TITLE
Support for it inside UIButton

### ISSUE THREAD

**Issue by prsolucoes** [tobihagemann/THLabel#47]
Hi,

This work inside UIButton on Title Label?

Thanks.

**Comment by tobihagemann** [comment]
I don't think that you can "just use" `THLabel` as the `titleLabel` of a `UIButton` because it is a read-only property. You could try to add a `THLabel` as a subview of the `UIButton` but that's just an uneducated guess from me.
